### DISCUSSION TITLE
Sierra-1 V1 spin data

### DISCUSSION THREAD

**Discussion by JLGF1** [pierreaubert/spinorama#258] [General]
Any plans to load the recent V1 spin data so it can be compared to V2?

**Comment by pierreaubert** [comment]
Hello,

Can you point me to the missing data?

Thanks

**Comment by JLGF1** [comment]
I guess you would need to contact the vendor for the raw data?  (the full V1 spin data on the web site has now been supplanted by the V2)

![Sierra-1 Klippel On-axis](https://github.com/pierreaubert/spinorama/assets/64571200/4917195b-27ec-472c-a3cd-8e6c608c1e61)

**Comment by pierreaubert** [comment]
Please contact Dave and ask him to send the data to me. He is usually very open and provides the data directly to me often.

**Reply by JLGF1** [reply]
Looks like Erin has the (new) data?  Does he typically share it?

![On-axis compared to S1](https://github.com/pierreaubert/spinorama/assets/64571200/eef5e0fa-12ab-478a-ae4d-3560ec22bf86)

**Comment by pierreaubert** [comment]
Let's make a deal, you find the data (all 72 measurements) and I integrate
it on the website.

Pierre

On Mon, Dec 11, 2023 at 11:40 PM JLGF1 ***@***.***> wrote:

> Looks like Erin has the (new) data?
>
> On-axis.compared.to.S1.jpg (view on web)
> <https://github.com/pierreaubert/spinorama/assets/64571200/eef5e0fa-12ab-478a-ae4d-3560ec22bf86>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/258#discussioncomment-7826419>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH2N35U2QQSODG6OUXTYI7G2TAVCNFSM6AAAAABAI4IHPCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQMRWGQYTS>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
--
PfA

**Reply by JLGF1** [reply]
So, you requested V2 data from Ascend but don't wish to request V1 data from Ascend?  That seems odd as you are the data aggregator, not me,  but whatever.

**Comment by pierreaubert** [comment]
I didn’t request the data for V2. What seems odd to me is that you expect
me to collect all this data for free. Is that too much to ask 10 min of
your time?

**Reply by JLGF1** [reply]
I don't recall asking you to do this for free.  Do you have a method to provide compensation for ten minutes of your time?

I guess it's probably obvious why the vendor offered up V2 data but not V1 data?
